ANNOUNCE

```diff
@@ -36,12 +36,12 @@ The cclib development team
 ———
 
 Summary of changes since last version:
-* New bridge to Psi4 (Felipe S. S. Schneider)
-* New attribute zpve for zero-point vibrational energy correction (kuriba)
-* New attributes for electric transition dipoles of electronic transitions (mwykes)
-* Support ccenergies and mpenergies in ORCA
-* Support grads in MOLCAS (Daniele Padula)
-* Support thermochemistry attributes in GAMESS (Mark Perri)
-* Support Mulliken atomspins in Gaussian (Peter St. John)
-* Support fuzzy matching of attribute in ccget script
-* Many other minor improvements and bug fixes from Felipe S. S. Schneider, Alex Maldonado, James E T Smith, Dustin Wheeler, shijunang, Waylon Peng and others
+* [GSOC2020] New method: bader calculates QTAIM partial charges (Minsik Cho)
```

Comment:
Can you make the same changes here that you made in `CHANGELOG`?

Reply:
Done, and in the docs changelog.

Reply:
Can you also use
```
[GSOC2020] New methods: Bader's QTAIM, Bickelhaupt, Stockholder, Hirshfeld, and DDEC6 partial charges (Minsik Cho)
```
to replace the top two lines?

Reply:
Done.